flipkart_scraper.py

```diff
@@ -0,0 +1,140 @@
+import requests  
+from urllib.request import urlopen as ureq
+from bs4 import BeautifulSoup as soup 
+import smtplib 
+import io  
+import csv 
+# Call the total script like : scrape_page_type_1(get_urls(4,6),6), where 4 specifies the dropdownselection and 6 is the number of pages
+
+# The front end can make use of the passing parameters
+# In the dropdown list, the developer can give the Words like Laptops, Mobiles, Cameras, Headphones, Air conditioners etc
+# Also the user can define the limit of number of pages to scrap [It is used in both functions]
+
+def get_urls(DropDownSelection,limit):
+  '''When get_url() is called, it makes a list of links according to the limit and selection
+  in order to forward to the scraper part
+  Argument1= DropDownSelection :: Selects the category from the frontend
+  Argument2= limit :: The limit for the number of web pages selected by user or by developer in frontend'''
```

Comment:
Please put the `"""` on the top and below the description. Also, I see you provided a docstring for the return statement below which is incorrect. I suggest to go with something like:

```python
def get_urls(DropDownSelection, limit):: 
    """ 
    Function to make a list of links according to the limit and selection.
  
    Parameters: 
    DropDownSelection (int): Selects the category from the frontend
    limit (int): The limit for the number of web pages selected by user or by developer in frontend
  
    Returns: 
    urlss (List): List names urlss that contains the list of links based on the given category.
    """
  
    return arg1 
```

Reply:
@btkcodedev please correct the docstring as suggested in the example